internal/project/framework/dbdrivers/mongo.go

```diff
@@ -56,24 +56,29 @@ func getAllMongoTenantDB(tenantCfgs []*TenantConnections) (map[uint64]*mongo.Cli
 			}
 		}
 
-		mongoCfg := cfgsMap["mongodb"]
-		userName := mongoCfg["username"].(string)
-		password := mongoCfg["password"].(string)
-
-		// IMPORTANT: If tenant database password is encrypted in master db config.
-		tenantDBPassPhraseKey := viper.GetString("database.tenant.secret")
-		if tenantDBPassPhraseKey != "" {
-			password, err = aes.MelonpanAESDecrypt(tenantDBPassPhraseKey, password)
-			if err != nil {
-				panic(err)
+		// IMPORTANT: Check the `mongodb` object exist in the Connections column or not.
+		if mongoCfg, ok := cfgsMap["mongodb"]; ok {
+			userName := mongoCfg["username"].(string)
+			password := mongoCfg["password"].(string)
+
+			// IMPORTANT: If tenant database password is encrypted in master db config.
+			tenantDBPassPhraseKey := viper.GetString("database.tenant.secret")
```

Comment:
Is there a situation that mongodb，mysql，redis are in the config at the same time? if in this time, mongodb, mysql, and redis must use the same password

Reply:
@gaoweiqiao-tre The answer is no :) . Because, in any framework the expectation is that you set password/username separately for  each of the database flavor.
Why? because different database has different default requirements. For example, mongodb default setting is no username and no password, mysql default setting is username = root and password can be empty. Redis default setting is no user name and no password but you can set password if you want however you can't set username for redis.

As a result it is not a good idea to maintain a single password properties for all kinds of db.

Reply:
I made a misstake, I thought "database.tenant.secret" is password 😄